R/get_source_expressions.R

```diff
@@ -169,7 +169,7 @@ get_source_expressions <- function(filename) {
       file_lines = source_file$lines,
       content = source_file$lines,
       full_parsed_content = parsed_content,
-      full_xml_parsed_content = if (!is.null(parsed_content)) tryCatch(xml2::read_xml(xmlparsedata::xml_parse_data(parsed_content)), error = function(e) NULL),
+      full_xml_parsed_content = if (is.null(parsed_content)) NULL else safe_parse_to_xml(parsed_content),
```

Comment:
Can we move the `NULL` check to `safe_parse_to_xml` as well?

Reply:
OK. I only didn't initially because the other usage doesn't have the `NULL` check (I assume because it's always non-`NULL` there, but wasn't sure enough to proceed).